WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
 			}
 			is_adult = yes
-			prestige = 5000
+			prestige = 2000
```

Comment:
The problem is that they never spanwn. They are useless because of insane Prestige numbers. They are unbalanced and have long been a mistake. It would be better for the titles not to exist at all then be totally useless.